flight/targets/omnibusf3/target-defs.mk

```diff
@@ -1,5 +1,6 @@
 # Add this board to the list of buildable boards
 ALL_BOARDS += omnibusf3
+NODEBUG_BOARDS += omnibusf3
```

Comment:
Maybe put a comment explaining why it's a NODEBUG board?

Reply:
I think something like this, where it's the same file, providing the same set of variables for a large number of boards--- that it's a waste to document the same thing in every one.

A disproportional amount of the time we do that, the comments end up becoming lies, too.

Reply:
Problem is non-regular maintainers usually aren't interested in anything other than their own issue on their own board, and they'll be lacking this information.

Reply:
Yah, it's all a case of relative risks.  Chance of doc helping someone vs. cost of keeping it mamy places and chance of it rotting and misleading.